setup.py

```diff
@@ -27,7 +27,10 @@
             "pytest_rts_specific_without_remap=pytest_rts.tests.evaluation.specific_without_remap:main",
             "pytest_rts_all_without_remap=pytest_rts.tests.evaluation.all_without_remap:main",
             "pytest_rts_collect=pytest_rts.collect:main",
-        ]
+        ],
```

Comment:
shall we remove the lines above?

Reply:
At least the collection of newly added tests and the existing evaluation code (which I don't know if we're ever going to use anymore) relies on those.

Reply:
let's remove at least `pytest_rts` and `pytest_rts_init`

Reply:
Removed all the ones not related to new test collection or evaluation. Also removed the code that they were pointing to.